packages/jaeger-ui/src/components/common/ExternalLinks.tsx

```diff
@@ -39,25 +39,22 @@ const LinkValue = (props: {
 );
 
 // export for testing
-export const linkValueList = (links: Link[]) => (
-  <Menu>
-    {links.map(({ text, url }, index) => (
-      // `index` is necessary in the key because url can repeat
-      // eslint-disable-next-line react/no-array-index-key
-      <Menu.Item key={`${url}-${index}`}>
-        <LinkValue href={url}>{text}</LinkValue>
-      </Menu.Item>
-    ))}
-  </Menu>
-);
+export const linkValueList = (links: Link[]) => {
+  const dropdownItems = links.map(({ text, url }, index) => ({
+    label: <LinkValue href={url}>{text}</LinkValue>,
+    key: `${url}-${index}`,
+  }));
+
+  return dropdownItems;
+};
 
 export default function ExternalLinks(props: ExternalLinksProps) {
```

Comment:
```suggestion
// ExternalLinks are displayed as a menu at the trace level.
// Example: https://github.com/jaegertracing/jaeger-ui/assets/94157520/7f0d84bc-c2fb-488c-9e50-1ec9484ea1e6
export default function ExternalLinks(props: ExternalLinksProps) {
```

Reply:
Is it correct that these external links are derived via link patterns?

Reply:
> Is it correct that these external links are derived via link patterns?

Here's the computation function in codebase.
https://github.com/jaegertracing/jaeger-ui/blob/797e0d80fe5523acb8e81484ea35ca046c071750/packages/jaeger-ui/src/model/link-patterns.tsx#L144-L164